configs/bootstrap.conf

```diff
@@ -44,6 +44,10 @@ java.arg.4=-Djava.net.preferIPv4Stack=true
 java.arg.5=-Dsun.net.http.allowRestrictedHeaders=true
 java.arg.6=-Djava.protocol.handler.pkgs=sun.net.www.protocol
 
+{{ if .Values.certManager.replaceDefaultTrustStore }}
+java.arg.7=-Djavax.net.ssl.trustStore=/opt/nifi/nifi-current/tls/truststore.jks
+{{/* if .Values.certManager.replaceDefaultTrustStore */}}{{ end }}
```

Comment:
is this a reminder of the condition at the end of the if block, or something that can be removed?

Reply:
Exactly right - I'm not a fan of how easy it is to lose track of nested Helm conditionals (e.g. https://github.com/cetic/helm-nifi/commit/2c5fdc0f7d0361f0a895aac8f7a02efd4a535416).